models/user/list.go

```diff
@@ -30,13 +30,19 @@ func (users UserList) GetTwoFaStatus() map[int64]bool {
 	for _, user := range users {
 		results[user.ID] = false // Set default to false
 	}
-	tokenMaps, err := users.loadTwoFactorStatus(db.GetEngine(db.DefaultContext))
-	if err == nil {
+
+	if tokenMaps, err := users.loadTwoFactorStatus(db.GetEngine(db.DefaultContext)); err == nil {
 		for _, token := range tokenMaps {
 			results[token.UID] = true
 		}
 	}
 
+	if credentialMaps, err := users.loadWebAuthnStatus(db.GetEngine(db.DefaultContext)); err == nil {
+		for _, cred := range credentialMaps {
+			results[cred.UserID] = true
+		}
+	}
+
```

Comment:
Given we're just using the UID here can't we just get the UIDs from the db instead of the full 2fa/webauthn credentials.

Given users can have more than one twofactor/authcredentails could we use  `DISTINCT` to just get unique uids out?